mandatory/2-function-creation.js

```diff
@@ -15,26 +17,56 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  if (totalPrice > 200) {
+return totalPrice * 0.9;
+} else {
+return totalPrice * 0.95;
+};
+}
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
-function printOddNumbers(limit) {}
+function printOddNumbers(limit) {
+  let number = 1;
+while (number <= limit) {
+if (number % 2 == 1) {
+console.log(number);
+}
+number++;
+};
+}
 
 /*
   Complete the buyTwoGetTheCheapestFree function: if user buys two items, the cheapest item will be free!
   The function should return the price to be paid once the discount is applied.
 */
-function buyTwoGetTheCheapestFree(price1, price2) {}
+function buyTwoGetTheCheapestFree(price1, price2) {
+  if (price1 > price2) {
+return price1;
+} else if (price2 > price1) {
+return price2;
+} else {
+  return price1||price2;
+};
+}
 
 /*
   Complete the function to determine if it is suitable for a person to register based on their age!
   - if the person is 12 or younger it should return "You Are Too Young To Register"
   - if the person is older than 12 and younger than 90 it should return "You Can Register"
   - if the person is 90 or older it should return "You Don't Need To Register"
 */
-function canRegister(age) {}
+function canRegister(age) {
+  if (age < 13) {
```

Comment:
Prefer using `<=` and `>=` in case the text is defined that way to avoid differences between the specification and the solution

Reply:
good point, thank you